service-binding/lambda/README.md

```diff
@@ -56,15 +56,15 @@ Apply a battery of `yaml` files to run the example.
 7. Trigger the function.
     The information and statistics about the Redis server appear in the logs of the function Pod.
     ```bash
-    kubeless function call redis-client -n $KYMA_EXAMPLE_NS
+     curl curl -ik https://redis-client.kyma.local
```

Comment:
The API defines redis-client-test as host
curl is doubled
It is fixed for minikube scenario... Maybe use a variable like $KYMA_DOMAIN